contracts/ReversibleICO.sol

```diff
@@ -329,33 +329,26 @@ contract ReversibleICO is IERC777Recipient {
         uint256 indexed _value
     );
 
-
     struct TotalsByStage {
         uint256 received;		    // msg.value
         uint256 returned;		    // received - accepted
         uint256 accepted;		    // lower than msg.value if maxCap already reached
         uint256 withdrawn;		    // withdrawn from current stage
         uint256 tokens_reserved;    // tokens bought in this stage
         uint256 tokens_awarded;	    // tokens already sent to the user in this stage
+        uint256 tokens_returned;	// tokens returned by participant to contract
     }
 
     struct Participant {
-
-        uint256 contributed_amount;
-        uint256 accepted_amount;
-        uint256 withdrawn_amount;
-        uint256 available_amount;
-        uint256 token_amount;
-        mapping ( uint16 => Contribution ) contributions;
-
         bool   whitelisted;
         uint16  contributionsCount;
         uint256 received;	        // msg.value
         uint256 returned;	        // received - accepted
         uint256 accepted;	        // lower than msg.value if maxCap already reached
         uint256 withdrawn;	        // cancel() / withdraw()
-        uint256 tokens_reserved;    // tokens bought in all stages
-        uint256 tokens_awarded;	    // tokens already sent to the user in all stages
+        uint256 tokens_reserved;    // total tokens bought in all stages
+        uint256 tokens_awarded;	    // total tokens already sent to the user in all stages
```

Comment:
Lets make sure we call users or participants the same everywhere.
I guess "user" makes it easier to understand. So replace all `participant`